Marvelys-Torres/tarea-1/index.html

```diff
@@ -14,54 +14,67 @@
     />
   </head>
   <body>
-    <hr />
-    <div class="logo">
-      <p>Perú</p>
-    </div>
-    <div class="heading">
-      <h1>AYO</h1>
-      <p>EL PARAÍSO ESCONDIDO</p>
-    </div>
-    <div class="grid-container">
-      <div class="content-1">
-        <p class="content-1-1">
-          Sus paisajes son tan peculiares como su <strong>Nombre.</strong> Este
-          valle se debe a la fundación de dos grandes familias: los Aguirre y
-          los Oviedo.
-        </p>
-        <p class="content-1-2">
-          Ven y descubre las maravillas de este hermoso lugar UBICADO EN
-          <strong>AREQUIPA</strong>
-        </p>
-        <p class="content-1-3">
-          <strong> Aquí encontrarás la laguna de MAMACOCHA,</strong> esta
-          hermosa laguna tiene aguas turquesas y en ellas nutrias!
-        </p>
+    <header>
+      <hr />
+      <div
```

Comment:
@MarvelysTorres we should not be using divs anymore, you can use section instead